substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVTypeRecord.java

```diff
@@ -357,6 +449,113 @@ public boolean equals(Object obj) {
         }
     }
 
+    @SuppressWarnings("unused")
+    static final class CVTypeModifierRecord extends CVTypeRecord {
+
+        private final int typeIndex;
+        private final short attrs;
+
+        CVTypeModifierRecord(int typeIndex, short attrs) {
+            super(LF_MODIFIER);
+            this.typeIndex = typeIndex;
+            this.attrs = attrs;
+        }
+
+        CVTypeModifierRecord(int typeIndex, boolean isConst, boolean isVolatile, boolean isUnaligned) {
+            this(typeIndex, (short) ((isConst ? 1 : 0) + (isVolatile ? 2 : 0) + (isUnaligned ? 4 : 0)));
+        }
+
+        @Override
+        public int computeContents(byte[] buffer, int initialPos) {
+            int pos = CVUtil.putInt(typeIndex, buffer, initialPos);
+            return CVUtil.putShort(attrs, buffer, pos);
+        }
+
+        @Override
+        public String toString() {
+            boolean isConst = (attrs & 0x0001) == 0x0001;
+            boolean isVolatile = (attrs & 0x0002) == 0x0002;
+            boolean isUnaligned = (attrs & 0x0004) == 0x0004;
+            return String.format("LF_MODIFIER 0x%04x attr=0x%x%s%s%s pointTo=0x%04x", getSequenceNumber(), attrs, isConst ? " const" : "", isVolatile ? " volatile" : "",
+                            isUnaligned ? " unaligned" : "", typeIndex);
+        }
+
+        @Override
+        public int hashCode() {
+            int h = type;
+            h = 31 * h + typeIndex;
+            h = 31 * h + attrs;
+            return h;
+        }
+
+        @Override
+        public boolean equals(Object obj) {
+            if (!super.equals(obj)) {
+                return false;
+            }
+            CVTypeModifierRecord other = (CVTypeModifierRecord) obj;
+            return this.typeIndex == other.typeIndex && this.attrs == other.attrs;
+        }
+    }
+
+    static final class CVTypeProcedureRecord extends CVTypeRecord {
```

Comment:
This is already unused. Given the above comments (https://github.com/oracle/graal/pull/5638#discussion_r2382022923, https://github.com/oracle/graal/pull/5638#discussion_r2382135062), I’d suggest removing it as well.